### ISSUE TITLE
Reimporting Your Board Code

### ISSUE THREAD

**Issue by GamertagIsAnt** [Firebottle/Firebot#160]
This wipes all the data you saved for your buttons, I wanted to change some names around it completely wiped all saved sounds, Images, ect. 

Can we get a update code section for the current board loaded so we don't see that issue please or when selecting add new board and importing the same code as a board already in the bot it doesn't just overwrite everything back to default Blankness :)

**Comment by ThePerry-2015** [comment]
You don't have to reimport you board if you do changes in the board within Mixed Interactive Studio.
Just open the Interactive Board in Firebot -> click on "Select a board" -> click on the name of the board and it will be updated with the new changes from Interactive Studio without touching any of the buttons already configured within Firebot.

Just tested this with my testing board, I added new buttons as well as changed names on some of the old buttons just to see what happened inside Firebot, all buttons that I had configured before I added new buttons in Studio as well as altering some names and spark costs on the old ones still has the same settings as they had before, nothing got removed it just changed button names and spark costs.

This was tested with Firebot 3.2.0, currently the latest released version available, freshly installed without copying anything over from an earlier setup just to be certain that nothing from an old setup came in the way of settings.
